jsonnet/rules.jsonnet

```diff
@@ -316,6 +316,19 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
               severity: 'info',
             },
           },
+          {
+            expr: 'avg_over_time((((count((max by (node) (up{job="kubelet",metrics_path="/metrics"} == 1) and max by (node) (kube_node_status_condition{condition="Ready",status="true"} == 1) and min by (node) (kube_node_spec_unschedulable == 0))) / scalar(count(min by (node) (kube_node_spec_unschedulable == 0))))))[5m:1s])',
```

Comment:
Why is an average the important key here? This query gives me result 1 on a cluster that has 10 nodes with 1 unready, is that correct?

Just curious about it that's all :D

Reply:
So in the last 5m if you had 1 node out of 10 unready for 1m, you'd get 0.9 + 0.2 (1 node unready for 1m out of 5).  If you scaled down and the node didn't go unready, you'd have 1.  We can estimate over time what the percent downtime was, but also work backward from node count to estimate how many nodes were down.

Until we move to direct write, the average also gives us a better rolling window of downtime with the current scrape-every-5m model - when I modelled it as instantaneous average we overreported the real average.